api/swagger.yaml

```diff
@@ -8045,7 +8045,7 @@ paths:
             - `label=key` or `label="key=value"` of an image label
             - `reference`=(`<image-name>[:<tag>]`)
             - `since`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
-          type: "string"
+            - `until=<timestamp>`
```

Comment:
Looks like `type: "string"` was dropped here; was that intentional or an accidental change?

Reply:
Sorry for that, it was accidental. I have pushed a fix.